imap_processing/tests/spice/test_geometry.py

```diff
@@ -151,6 +151,29 @@ def test_frame_transform(et_strings, position, from_frame, to_frame, furnish_ker
             np.testing.assert_allclose(test_result, spice_result, atol=1e-12)
 
 
+@pytest.mark.parametrize(
+    "spice_frame",
+    [
+        SpiceFrame.IMAP_DPS,
+        SpiceFrame.IMAP_SPACECRAFT,
+        SpiceFrame.ECLIPJ2000,
+    ],
+)
+@pytest.mark.parametrize(
+    "position",
+    [
+        np.array([1, 0, 0]),
+        np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]]),
+        np.random.rand(10, 3),
+    ],
+)
+def test_frame_transform_same_frame(position, spice_frame):
+    """Test that frame_transform returns position when input/output frames are same."""
+    position = np.array([1, 0, 0])
+    result = frame_transform(0, position, spice_frame, spice_frame)
+    np.testing.assert_array_equal(result, position)
```

Comment:
If you want this to be a true no-op, this is where an `is` check would be good to make sure they are the same object.
```suggestion
    assert result is position
```

Reply:
Ah, that makes sense. Thanks!